### PR TITLE
Update files for the v4.0.24 release

### PR DESCRIPTION
**Pull request by lucas-hay**

### CODE REVIEW

File: site/pages/docs/versions/v4.0/v4.0-changelog-fr.hbs [hunk 1]
```diff
@@ -4,34 +4,32 @@
...
 	"altLangPrefix": "v4.0-changelog",
-	"dateModified": "2016-11-18"
+	"dateModified": "2017-02-22"
 }
 ---
 <p>Voir <a href="../version-info-fr.html">Contrôle des versions</a> pour la signification de chaque numéro de version.</p>
 <span class="wb-prettify all-pre"></span>
 <section>
-	<h2>v4.0.23</h2>
+	<h2>v4.0.24</h2>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/v4.0/v4.0-changelog-fr.hbs]: Shopuldn't this be a new section instead of replacing this one?

**Review by LaurentGoderre — CHANGES_REQUESTED** [review]

File: site/pages/docs/versions/v4.0/v4.0-changelog-fr.hbs (hunk 1 above)
Comment (lucas-hay) [inline site/pages/docs/versions/v4.0/v4.0-changelog-fr.hbs]: Yep.
Fixed

**Review by LaurentGoderre — APPROVED** [review]

**Comment by LaurentGoderre** [comment]
I will merge when the build pass. In the mean time, can you restart the latest build on TRavis for every theme?

**Comment by MaximMartel** [comment]
All restarted @LaurentGoderre

**Comment by lucas-hay** [comment]
@MaximMartel Thanks!

**Pull request merged**